WooCommerce/Classes/ViewRelated/Feature Highlight/Tooltip.swift

```diff
@@ -0,0 +1,420 @@
+import UIKit
+
+final class Tooltip: UIView {
+    static let arrowWidth: CGFloat = 17
+
+    private enum Constants {
+        static let leadingIconUnicode = "✨"
+        static let cornerRadius: CGFloat = 4
+        static let arrowTipYLength: CGFloat = 8
+        static let arrowTipYControlLength: CGFloat = 9
+        static let invertedTooltipBackgroundColor = UIColor(
+            light: UIColor.systemGray5.color(for: UITraitCollection(userInterfaceStyle: .dark)),
+            dark: .white
+        )
```

Comment:
nit: can this color be in `UIColor+Tooltip`?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10005/commits/0f3612c619f06cc654c9510719f767909fbac785